src/main/java/com/distillery/intentsender/adb/AdbHelper.java

```diff
@@ -249,13 +249,19 @@ private String getFullCommand(@NotNull CommandType type, String action, String d
 			}
 		}
 		if (flags != null && flags.size() > 0) {
-			for (IntentFlags flag : flags) {
-				builder.append(flag.getCommand());
-			}
+			flags.forEach(flag -> {
```

Comment:
Nit: I think loop syntax is more readable and easier to debug (you can simply use F8 for stepping over the loop).

Reply:
As of debugging, F8 works perfect:
https://user-images.githubusercontent.com/3169756/117585226-f6eda400-b119-11eb-82d6-399aa64abcaa.mov

In general, there are various talks on this subject. For example, here are couple opinions from the same thread on stackoverflow: [1](https://stackoverflow.com/a/16635489/6556528) and [2](https://stackoverflow.com/a/22502206/6556528). I can't say it matters that much for me, but I find some points in the second comment pretty much reasonable. I am ready to continue this discussion, but I just feel that it does not worth the time in this case. Unless you want to have a discussion with some 🍺 . So, if you do not mind, I would leave this change as is.

Reply:
Yep, sure, that was just my opinion, not a request.